Error building the project for first time - "Unable to resolve .vcxproj file for .NetFramework"

<!-- 
🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨

I ACKNOWLEDGE THE FOLLOWING BEFORE PROCEEDING:
1. If I delete this entire template and go my own path, the core team may close my issue without further explanation or engagement.
2. If I list multiple bugs/concerns in this one issue, the core team may close my issue without further explanation or engagement.
3. If I write an issue that has many duplicates, the core team may close my issue without further explanation or engagement (and without necessarily spending time to find the exact duplicate ID number).
4. If I leave the title incomplete when filing the issue, the core team may close my issue without further explanation or engagement.
5. If I file something completely blank in the body, the core team may close my issue without further explanation or engagement.

All good? Then proceed!
-->

<!--
This bug tracker is monitored by Windows Terminal development team and other technical folks.

**Important: When reporting BSODs or security issues, DO NOT attach memory dumps, logs, or traces to Github issues**.
Instead, send dumps/traces to secure@microsoft.com, referencing this GitHub issue.

If this is an application crash, please also provide a Feedback Hub submission link so we can find your diagnostic data on the backend. Use the category "Apps > Windows Terminal (Preview)" and choose "Share My Feedback" after submission to get the link.

Please use this form and describe your issue, concisely but precisely, with as much detail as possible.

-->

# Environment
Microsoft Windows [Version 10.0.18362.449]

# Steps to reproduce

<!-- A description of how to trigger this bug. -->

[Prerequisites](https://github.com/microsoft/terminal#prerequisites)

1. Open Terminal as administrator (cmd)
2. Run `git submodule update --init --recursive`
3. Run `tools\razzle`
4. Run `bcz`


# Expected behavior

Ability to completely build the terminal using Razzle `bcz` command

# Actual behavior

Error running `tools\razzle`:
```
Setting up dev environment...

Errors in C:\projects\terminal\src\cascadia\WpfTerminalControl\WpfTerminalControl.csproj
    Unable to resolve 'C:\projects\terminal\src\cascadia\PublicTerminalCore\PublicTerminalCore.vcxproj' for '.NETFramework,Version=v4.7.2'.
The dev environment is ready to go!
````

Continuing to run `bcz` leaves me with:
```
Skipping building appx...
Performing nuget restore...
MSBuild auto-detection: using msbuild version '16.3.2.50909' from 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\bin'.
All packages listed in packages.config are already installed.
Restoring packages for C:\projects\terminal\src\cascadia\WpfTerminalControl\WpfTerminalControl.csproj...
Unable to resolve 'C:\projects\terminal\src\cascadia\PublicTerminalCore\PublicTerminalCore.vcxproj' for '.NETFramework,Version=v4.7.2'.
Committing restore...
Lock file has not changed. Skipping lock file write. Path: C:\projects\terminal\src\cascadia\WpfTerminalControl\obj\project.assets.json
Restore failed in 586.02 ms for C:\projects\terminal\src\cascadia\WpfTerminalControl\WpfTerminalControl.csproj.        
Errors in C:\projects\terminal\src\cascadia\WpfTerminalControl\WpfTerminalControl.csproj
    Unable to resolve 'C:\projects\terminal\src\cascadia\PublicTerminalCore\PublicTerminalCore.vcxproj' for '.NETFramework,Version=v4.7.2'.

NuGet Config files used:
    C:\projects\terminal\NuGet.Config
    C:\Users\Bradl\AppData\Roaming\NuGet\NuGet.Config
    C:\Program Files (x86)\NuGet\Config\Microsoft.VisualStudio.Offline.config

Feeds used:
    https://api.nuget.org/v3/index.json
    https://terminalnuget.blob.core.windows.net/feed/index.json
    C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\
"C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\MSBuild.exe" C:\projects\terminal\OpenConsole.sln /t:"Clean,Build" /m /p:Configuration=Debug /p:Platform=x64 /p:AppxBundle=false
Starting build...
Microsoft (R) Build Engine version 16.3.2+e481bbf88 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

Build started 10/31/2019 11:27:33 PM.
     1>Project "C:\projects\terminal\OpenConsole.sln" on node 1 (Clean,Build target(s)).
     1>ValidateSolutionConfiguration:
         Building solution configuration "Debug|x64".
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\cascadia\CascadiaPacka
       ge\CascadiaPackage.wapproj" (2) on node 1 (Clean,Build target(s)).
     2>C:\projects\terminal\src\cascadia\CascadiaPackage\CascadiaPackage.wapproj : error MSB4057: The target "Clean,Bu
       ild" does not exist in the project.
     2>Done Building Project "C:\projects\terminal\src\cascadia\CascadiaPackage\CascadiaPackage.wapproj" (Clean,Build
       target(s)) -- FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\propslib\propslib.vcxp
       roj" (4) on node 2 (Clean,Build target(s)).
     4>C:\projects\terminal\src\propslib\propslib.vcxproj : error MSB4057: The target "Clean,Build" does not exist in
       the project.
     4>Done Building Project "C:\projects\terminal\src\propslib\propslib.vcxproj" (Clean,Build target(s)) -- FAILED.        1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\tsf\tsf.vcxproj" (3) o        n node 3 (Clean,Build target(s)).
     3>C:\projects\terminal\src\tsf\tsf.vcxproj : error MSB4057: The target "Clean,Build" does not exist in the projec        t.
     3>Done Building Project "C:\projects\terminal\src\tsf\tsf.vcxproj" (Clean,Build target(s)) -- FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\renderer\base\lib\base        .vcxproj" (5) on node 4 (Clean,Build target(s)).
     5>C:\projects\terminal\src\renderer\base\lib\base.vcxproj : error MSB4057: The target "Clean,Build" does not exis        t in the project.
     5>Done Building Project "C:\projects\terminal\src\renderer\base\lib\base.vcxproj" (Clean,Build target(s)) -- FAIL        ED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\terminal\parser\ft_fuz        zer\VTCommandFuzzer.vcxproj" (6) on node 8 (Clean,Build target(s)).
     6>C:\projects\terminal\src\terminal\parser\ft_fuzzer\VTCommandFuzzer.vcxproj : error MSB4057: The target "Clean,B        uild" does not exist in the project.
     6>Done Building Project "C:\projects\terminal\src\terminal\parser\ft_fuzzer\VTCommandFuzzer.vcxproj" (Clean,Build         target(s)) -- FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\terminal\adapter\ut_ad        apter\Adapter.UnitTests.vcxproj" (7) on node 7 (Clean,Build target(s)).
     7>C:\projects\terminal\src\terminal\adapter\ut_adapter\Adapter.UnitTests.vcxproj : error MSB4057: The target "Cle        an,Build" does not exist in the project.
     7>Done Building Project "C:\projects\terminal\src\terminal\adapter\ut_adapter\Adapter.UnitTests.vcxproj" (Clean,B        uild target(s)) -- FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\terminal\parser\ut_par        ser\Parser.UnitTests.vcxproj" (8) on node 6 (Clean,Build target(s)).
     8>C:\projects\terminal\src\terminal\parser\ut_parser\Parser.UnitTests.vcxproj : error MSB4057: The target "Clean,        Build" does not exist in the project.
     8>Done Building Project "C:\projects\terminal\src\terminal\parser\ut_parser\Parser.UnitTests.vcxproj" (Clean,Buil        d target(s)) -- FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\propsheet\propsheet.vc        xproj" (9) on node 2 (Clean,Build target(s)).
     9>C:\projects\terminal\src\propsheet\propsheet.vcxproj : error MSB4057: The target "Clean,Build" does not exist i        n the project.
     9>Done Building Project "C:\projects\terminal\src\propsheet\propsheet.vcxproj" (Clean,Build target(s)) -- FAILED.      1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\renderer\gdi\lib\gdi.v
       cxproj" (10) on node 5 (Clean,Build target(s)).
    10>C:\projects\terminal\src\renderer\gdi\lib\gdi.vcxproj : error MSB4057: The target "Clean,Build" does not exist         in the project.
    10>Done Building Project "C:\projects\terminal\src\renderer\gdi\lib\gdi.vcxproj" (Clean,Build target(s)) -- FAILED        .
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\server\lib\server.vcxp        roj" (11) on node 3 (Clean,Build target(s)).
    11>C:\projects\terminal\src\server\lib\server.vcxproj : error MSB4057: The target "Clean,Build" does not exist in         the project.
    11>Done Building Project "C:\projects\terminal\src\server\lib\server.vcxproj" (Clean,Build target(s)) -- FAILED.        1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\tools\vtapp\VTApp.cspr        oj" (12) on node 4 (Clean,Build target(s)).
    12>C:\projects\terminal\src\tools\vtapp\VTApp.csproj : error MSB4057: The target "Clean,Build" does not exist in t        he project.
    12>Done Building Project "C:\projects\terminal\src\tools\vtapp\VTApp.csproj" (Clean,Build target(s)) -- FAILED.         1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\tools\fontlist\FontLis        t.vcxproj" (13) on node 7 (Clean,Build target(s)).
    13>C:\projects\terminal\src\tools\fontlist\FontList.vcxproj : error MSB4057: The target "Clean,Build" does not exi        st in the project.
    13>Done Building Project "C:\projects\terminal\src\tools\fontlist\FontList.vcxproj" (Clean,Build target(s)) -- FAI        LED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\tools\nihilist\Nihilis        t.vcxproj" (14) on node 8 (Clean,Build target(s)).
    14>C:\projects\terminal\src\tools\nihilist\Nihilist.vcxproj : error MSB4057: The target "Clean,Build" does not exi        st in the project.
    14>Done Building Project "C:\projects\terminal\src\tools\nihilist\Nihilist.vcxproj" (Clean,Build target(s)) -- FAI        LED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\tools\scratch\Scratch.        vcxproj" (16) on node 6 (Clean,Build target(s)).
    16>C:\projects\terminal\src\tools\scratch\Scratch.vcxproj : error MSB4057: The target "Clean,Build" does not exist
        in the project.
    16>Done Building Project "C:\projects\terminal\src\tools\scratch\Scratch.vcxproj" (Clean,Build target(s)) -- FAILE        D.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\tools\buffersize\buffe        rsize.vcxproj" (15) on node 2 (Clean,Build target(s)).
    15>C:\projects\terminal\src\tools\buffersize\buffersize.vcxproj : error MSB4057: The target "Clean,Build" does not         exist in the project.
    15>Done Building Project "C:\projects\terminal\src\tools\buffersize\buffersize.vcxproj" (Clean,Build target(s)) --         FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\tools\closetest\CloseT        est.vcxproj" (17) on node 3 (Clean,Build target(s)).
    17>C:\projects\terminal\src\tools\closetest\CloseTest.vcxproj : error MSB4057: The target "Clean,Build" does not e
       xist in the project.
    17>Done Building Project "C:\projects\terminal\src\tools\closetest\CloseTest.vcxproj" (Clean,Build target(s)) -- F
       AILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\terminal\parser\ft_fuz        zwrapper\FuzzWrapper.vcxproj" (18) on node 1 (Clean,Build target(s)).
    18>C:\projects\terminal\src\terminal\parser\ft_fuzzwrapper\FuzzWrapper.vcxproj : error MSB4057: The target "Clean,
       Build" does not exist in the project.
    18>Done Building Project "C:\projects\terminal\src\terminal\parser\ft_fuzzwrapper\FuzzWrapper.vcxproj" (Clean,Buil
       d target(s)) -- FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\renderer\vt\lib\vt.vcx        proj" (19) on node 4 (Clean,Build target(s)).
    19>C:\projects\terminal\src\renderer\vt\lib\vt.vcxproj : error MSB4057: The target "Clean,Build" does not exist in         the project.
    19>Done Building Project "C:\projects\terminal\src\renderer\vt\lib\vt.vcxproj" (Clean,Build target(s)) -- FAILED.       1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\types\lib\types.vcxpro        j" (20) on node 8 (Clean,Build target(s)).
    20>C:\projects\terminal\src\types\lib\types.vcxproj : error MSB4057: The target "Clean,Build" does not exist in th
       e project.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\tools\echokey\ConEchoK        ey.vcxproj" (21) on node 7 (Clean,Build target(s)).
    21>C:\projects\terminal\src\tools\echokey\ConEchoKey.vcxproj : error MSB4057: The target "Clean,Build" does not ex        ist in the project.
    20>Done Building Project "C:\projects\terminal\src\types\lib\types.vcxproj" (Clean,Build target(s)) -- FAILED.         21>Done Building Project "C:\projects\terminal\src\tools\echokey\ConEchoKey.vcxproj" (Clean,Build target(s)) -- FA        ILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\renderer\vt\ut_lib\vt.        unittest.vcxproj" (22) on node 6 (Clean,Build target(s)).
    22>C:\projects\terminal\src\renderer\vt\ut_lib\vt.unittest.vcxproj : error MSB4057: The target "Clean,Build" does         not exist in the project.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\renderer\dx\lib\dx.vcx        proj" (23) on node 3 (Clean,Build target(s)).
    23>C:\projects\terminal\src\renderer\dx\lib\dx.vcxproj : error MSB4057: The target "Clean,Build" does not exist in         the project.
    23>Done Building Project "C:\projects\terminal\src\renderer\dx\lib\dx.vcxproj" (Clean,Build target(s)) -- FAILED.      22>Done Building Project "C:\projects\terminal\src\renderer\vt\ut_lib\vt.unittest.vcxproj" (Clean,Build target(s))
        -- FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\buffer\out\lib\buffero        ut.vcxproj" (24) on node 2 (Clean,Build target(s)).
    24>C:\projects\terminal\src\buffer\out\lib\bufferout.vcxproj : error MSB4057: The target "Clean,Build" does not ex        ist in the project.
    24>Done Building Project "C:\projects\terminal\src\buffer\out\lib\bufferout.vcxproj" (Clean,Build target(s)) -- FA
       ILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\interactivity\base\lib        \InteractivityBase.vcxproj" (25) on node 5 (Clean,Build target(s)).
    25>C:\projects\terminal\src\interactivity\base\lib\InteractivityBase.vcxproj : error MSB4057: The target "Clean,Bu        ild" does not exist in the project.
    25>Done Building Project "C:\projects\terminal\src\interactivity\base\lib\InteractivityBase.vcxproj" (Clean,Build         target(s)) -- FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\pkg\appx\OpenConsolePackag        e.wapproj" (26) on node 8 (Clean,Build target(s)).
    26>C:\projects\terminal\pkg\appx\OpenConsolePackage.wapproj : error MSB4057: The target "Clean,Build" does not exi        st in the project.
    26>Done Building Project "C:\projects\terminal\pkg\appx\OpenConsolePackage.wapproj" (Clean,Build target(s)) -- FAI        LED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\internal\internal.vcxp        roj" (27) on node 3 (Clean,Build target(s)).
    27>C:\projects\terminal\src\internal\internal.vcxproj : error MSB4057: The target "Clean,Build" does not exist in         the project.
    27>Done Building Project "C:\projects\terminal\src\internal\internal.vcxproj" (Clean,Build target(s)) -- FAILED.        1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\cascadia\PublicTermina        lCore\PublicTerminalCore.vcxproj" (28) on node 2 (Clean,Build target(s)).
    28>C:\projects\terminal\src\cascadia\PublicTerminalCore\PublicTerminalCore.vcxproj : error MSB4057: The target "Cl        ean,Build" does not exist in the project.
    28>Done Building Project "C:\projects\terminal\src\cascadia\PublicTerminalCore\PublicTerminalCore.vcxproj" (Clean,        Build target(s)) -- FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\types\ut_types\Types.U        nit.Tests.vcxproj" (29) on node 6 (Clean,Build target(s)).
    29>C:\projects\terminal\src\types\ut_types\Types.Unit.Tests.vcxproj : error MSB4057: The target "Clean,Build" does         not exist in the project.
    29>Done Building Project "C:\projects\terminal\src\types\ut_types\Types.Unit.Tests.vcxproj" (Clean,Build target(s)        ) -- FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\cascadia\UnitTests_Ter        minalCore\UnitTests.vcxproj" (30) on node 7 (Clean,Build target(s)).
    30>C:\projects\terminal\src\cascadia\UnitTests_TerminalCore\UnitTests.vcxproj : error MSB4057: The target "Clean,B        uild" does not exist in the project.
    30>Done Building Project "C:\projects\terminal\src\cascadia\UnitTests_TerminalCore\UnitTests.vcxproj" (Clean,Build         target(s)) -- FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\cascadia\TerminalSetti        ngs\TerminalSettings.vcxproj" (31) on node 4 (Clean,Build target(s)).
    31>C:\projects\terminal\src\cascadia\TerminalSettings\TerminalSettings.vcxproj : error MSB4057: The target "Clean,        Build" does not exist in the project.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\winconpty\winconpty.vc        xproj" (32) on node 8 (Clean,Build target(s)).
    32>C:\projects\terminal\src\winconpty\winconpty.vcxproj : error MSB4057: The target "Clean,Build" does not exist i        n the project.
    31>Done Building Project "C:\projects\terminal\src\cascadia\TerminalSettings\TerminalSettings.vcxproj" (Clean,Buil        d target(s)) -- FAILED.
    32>Done Building Project "C:\projects\terminal\src\winconpty\winconpty.vcxproj" (Clean,Build target(s)) -- FAILED.      1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\cascadia\TerminalConne        ction\TerminalConnection.vcxproj" (33) on node 1 (Clean,Build target(s)).
    33>C:\projects\terminal\src\cascadia\TerminalConnection\TerminalConnection.vcxproj : error MSB4057: The target "Cl        ean,Build" does not exist in the project.
    33>Done Building Project "C:\projects\terminal\src\cascadia\TerminalConnection\TerminalConnection.vcxproj" (Clean,
       Build target(s)) -- FAILED.
     1>Project "C:\projects\terminal\OpenConsole.sln" (1) is building "C:\projects\terminal\src\cascadia\WpfTerminalCo
       ntrol\WpfTerminalControl.csproj" (34) on node 5 (Clean,Build target(s)).
    34>C:\projects\terminal\src\cascadia\WpfTerminalControl\WpfTerminalControl.csproj : error MSB4057: The target "Cle        an,Build" does not exist in the project.
    34>Done Building Project "C:\projects\terminal\src\cascadia\WpfTerminalControl\WpfTerminalControl.csproj" (Clean,B        uild target(s)) -- FAILED.
     1>Done Building Project "C:\projects\terminal\OpenConsole.sln" (Clean,Build target(s)) -- FAILED.

Build FAILED.
```

So I was able to get rid of the top error:

`Errors in C:\projects\terminal\src\cascadia\WpfTerminalControl\WpfTerminalControl.csproj
    Unable to resolve 'C:\projects\terminal\src\cascadia\PublicTerminalCore\PublicTerminalCore.vcxproj' for '.NETFramework,Version=v4.7.2'.`

But all the following errors still remain as posted above. Each some form of:

`error MSB4057: The target "Clean,Build" does not exist in the project.`

Not sure what is causing this? Is there a specific directory I should only be building?

--- a/tools/bcz.cmd
+++ b/tools/bcz.cmd
@@ -18,7 +18,7 @@ if (%_LAST_BUILD_CONF%)==() (
     set _LAST_BUILD_CONF=%DEFAULT_CONFIGURATION%
 )
 
-set _MSBUILD_TARGET=Clean,Build
+set _MSBUILD_TARGET=Clean;Build
 set _EXCLUSIVE=
 set _APPX_ARGS=
 

